src/main/kotlin/com/hedvig/accountservice/accounts/model/AccountManager.kt

```diff
@@ -58,14 +59,18 @@ class AccountManager @Autowired constructor(
   fun getNextChargeEstimation(memberId: String): ChargeEstimationDto {
     val monthToCheckMaybe = getMonthForEstimation(memberId)
     val currentBalance = getCurrentBalance(memberId)
+    val marketInfo = productPricingService.getMarketInfoForMember(memberId)
```

Comment:
To make the methods more easily readable
```suggestion
    val currency = productPricingService.getMarketInfoForMember(memberId).preferredCurrency
```